components/LoginModal.vue

```diff
@@ -172,11 +191,17 @@ export default {
     const isForgotten = ref(false)
     const rememberMe = ref(false)
     const createAccount = ref(false)
+    const isThankYouAfterForgotten = ref(false)
```

Comment:
instead of isThankYouAfterForgotten can we give some more meaning full name?

Reply:
Done.